testbook/client.py

```diff
@@ -9,10 +9,42 @@
 from .testbooknode import TestbookNode
 
 
+class TestbookCallable:
+    def __init__(self, tb, name):
+        self.tb: TestbookNotebookClient = tb
+        self.name: str = name
+
+    def __repr__(self):
+        return f"<TestbookCallable name='{self.name}'>"
+
+    def __call__(self, *args, **kwargs):
+        args_str = ', '.join(map(repr, args)) if args else ''
```

Comment:
Repr isn't the right thing to use here. I'd use json parsing or papermill's translator style conversion

Reply:
e.g. `args_list = PythonTranslator.translate(args)`